File: cosmos/Cosmos-API/src/main/java/com/planbtech/cosmos/model/entites/Scholarity.java
package com.planbtech.cosmos.model.entites;

import com.planbtech.cosmos.model.Interfaces.IScholarity;
import jakarta.persistence.*;

@Entity(name = "scholarity")
public class Scholarity implements IScholarity {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "scholarity_id")
    private Long id;

    private String scholarity;

    @Override
    public Long getId() {
        return this.id;
    }

    @Override
    public void setId(Long id) {
        this.id = id;
    }

    @Override
    public String getScholarity() {
        return this.scholarity;
    }

    @Override
    public void setScholarity(String scholarity) {
        this.scholarity = scholarity;
    }
}

File: cosmos/ProConnect/src/app/Class/scholarity.ts
import { Ischolarity } from "../interfaces/ischolarity";

export class Scholarity implements Ischolarity{
  scholarity: string = '';
  id: number = 0;


  public  getId(): number { return this.id}
  public setId(id: number): void {this.id = id}

  public getScholarityTitle(): string {return this.scholarity}
  public setScholarityTitle(scholarity: string): void {this.scholarity = scholarity}
}
